dev/AnimatedIcon/AnimatedIcon.cpp

```diff
@@ -444,6 +455,33 @@ void AnimatedIcon::OnSourcePropertyChanged(const winrt::DependencyPropertyChange
     }
 }
 
+void AnimatedIcon::UpdateMirrorTransform()
+{
+    auto const scaleTransform = [this]()
+    {
+        if (!m_scaleTransform)
+        {
+            // Initialize the scale transform that will be used for mirroring and the
+            // render transform origin as center in order to have the icon mirrored in place.
+            winrt::Windows::UI::Xaml::Media::ScaleTransform scaleTransform;
+            scaleTransform.CenterX(ActualWidth() / 2);
+            scaleTransform.CenterY(ActualHeight() / 2);
+
+            RenderTransform(scaleTransform);
```

Comment:
Can this use RenderTransformOrigin = 0.5,0.5 instead of needing to watch for layout changes and use the ActualWidth/Height? (that's what FontIcon seems to be doing)

Reply:
Yeah, thanks I was looking for this and couldn't find it..